six/modules/c++/six.sicd/tests/test_streaming_write.cpp

```diff
@@ -1,10 +1,10 @@
 /* =========================================================================
```

Comment:
I'm getting a compile error at line 154 on Linux. I think it doesn't like getting `std::auto_ptr<six::sicd::ComplexData>` instead of `std::auto_ptr<six::Data>`.

   ```
 ../six/modules/c++/six.sicd/tests/test_streaming_write.cpp: In instantiation of ‘void {anonymous}::Tester<DataTypeT>::normalWrite() [with DataTypeT = fl     oat]’:
    ../six/modules/c++/six.sicd/tests/test_streaming_write.cpp:87:21:   required from ‘{anonymous}::Tester<DataTypeT>::Tester(const std::vector<std::basic_string<char> >&, bool, size_t) [with DataTypeT = float; size_t = long unsigned int]’
../six/modules/c++/six.sicd/tests/test_streaming_write.cpp:304:76:   required from ‘bool {anonymous}::doTests(const std::vector<std::basic_string<char> >&, bool, size_t) [with DataTypeT = float; size_t = long unsigned int]’
../six/modules/c++/six.sicd/tests/test_streaming_write.cpp:317:70:   required from here
../six/modules/c++/six.sicd/tests/test_streaming_write.cpp:154:5: error: no matching function for call to ‘std::auto_ptr<six::Data>::auto_ptr(std::auto_ptr<six::Data>)’
    mContainer->addData(createData<DataTypeT>(mDims));

```

Reply:
Ahh that's lame (Visual Studio figures it out), but thanks for letting me know - releasing the `auto_ptr` now so it can just do a simple pointer conversion (container will still take ownership).